src/hooks/useSwitch/__stories__/useSwitch.stories.mdx

```diff
@@ -0,0 +1,116 @@
+import { Story, Canvas, Meta } from "@storybook/addon-docs";
+import { FunctionArgument, FunctionArguments } from "../../../storybook";
+import useSwitch from "../index";
+import Toggle from "../../../components/Toggle/Toggle";
+import Typography from "../../../components/Typography/Typography";
+import styles from "./useSwitch.stories.module.scss";
+
+<Meta title="Hooks/useSwitch" />
+
+# useSwitch
+
+- [Overview](#overview)
+- [Variants](#variants)
+- [Arguments](#arguments)
+- [Returns](#returns)
+- [Feedback](#feedback)
+
+## Overview
+
+Hook for controlling boolean state on components, e.g. Toggle, by exposing state and a handler
+
+<Canvas>
+  <Story name="Overview">
+    {() => {
+      const { isChecked, onChange } = useSwitch();
+      return (
+        <div className={styles.switchContainer}>
+          <Toggle onChange={onChange} isSelected={isChecked} />
+          <Typography>
```

Comment:
Typography is internal component, use Text instead (if you want to use the code format I think it's also not needed)